pennylane/qchem/convert.py

```diff
@@ -414,7 +414,7 @@ def _excited_configurations(electrons, orbitals, excitation):
     r"""Generate excited configurations from a Hartree-Fock reference state.
 
     This function generates excited configurations in the form of integers representing a binary
-    string, e.g., :math:`|1 1 0 1 0 0 \rangle is represented by :math:`int('110100', 2) = 52`.
+    string, e.g., :math:`|1 1 0 1 0 0 \rangle is represented by ``int('110100', 2) = 52``.
```

Comment:
This looks weird ... Are we sure that we want "is represented" to render as math text as well? I would intuitively expect the fix to be to add a quote after \rangle. Feel free to resolve this if that still doesn't fix the issue.

```suggestion
    string, e.g., :math:`|1 1 0 1 0 0 \rangle` is represented by :math:`int('110100', 2) = 52`.
```